pkg/local_object_storage/engine/put.go

```diff
@@ -16,6 +16,10 @@ import (
 // PutPrm groups the parameters of Put operation.
 type PutPrm struct {
 	obj *objectSDK.Object
+
+	binSet bool
```

Comment:
`binSet` == `objBin != nil`?

Reply:
`binSet` prevents duality b/w uncalled and called with nil cases

Reply:
> called with nil cases

is it valid?

Reply:
depends on the caller. This is his responsibility, docs say that

Reply:
undesired complexity to me. calling with empty bytes ends with no error (i guess) while that is incorrect for sure